ansible/roles/cdh_common/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
   yum: name=hadoop-conf-pseudo state=present
 
 - name: copy /etc/hadoop/conf.pseudo to /etc/hadoop/conf.mycluster
-  command: cp -R -p /etc/hadoop/conf.pseudo /etc/hadoop/conf.mycluster creates=/etc/hadoop/conf.mycluster
+  copy: src=/etc/hadoop/conf.pseudo dest=/etc/hadoop/conf.mycluster directory_mode=yes remote_src=True
```

Comment:
http://docs.ansible.com/ansible/copy_module.html

`directory_mode`:
> When doing a recursive copy set the mode for the directories. 

`remote_src`:
> Currently remote_src does not support recursive copying.

So, you can't use `directory_mode=yes` and `remote_src=True` at the same time.